json/json.mbt

```diff
@@ -121,6 +121,25 @@ pub fn stringify(self : JsonValue) -> String {
   buf.to_string()
 }
 
+fn escape(str : String) -> String {
+  let buf = Buffer::new()
+  for i = 0; i < str.length(); i = i + 1 {
+    let c = str[i]
+    match c {
+      '"' | '\\' | '/' => {
+        buf.write_char('\\')
+        buf.write_char(c)
+      }
+      '\n' => buf.write_string("\\n")
+      '\r' => buf.write_string("\\r")
+      '\b' => buf.write_string("\\b")
+      '\t' => buf.write_string("\\t")
```

Comment:
https://stackoverflow.com/questions/19176024/how-to-escape-special-characters-in-building-a-json-string

According to this.
Probably missing `\f` and `\u`?

Reply:
Thanks!